packages/angular/projects/vcd/plugin-builders/src/lib/base/index.ts

```diff
@@ -47,239 +30,242 @@ export const defaultExternals = {
-                vendor: {
-                    test: filterRuntimeModules(options),
-                    name: nameVendorFile(config, options, pluginLibsBundles, manifest, manifestJsonPath),
-                    enforce: true,
-                },
-                // All other vendors go in common.js file
-                common: {
-                    test: (mod) => {
-                        if (!mod.context) {
-                            return false;
-                        }
-
-                        if (!Object.keys(options.librariesConfig).some((key) => {
-                                return mod.context.includes(key);
-                            })) {
-                            return true;
-                        }
-                        return false;
-                    },
-                    name: 'common',
-                    reuseExistingChunk: true,
-                    enforce: true,
-                }
-            },
+        /**
+         * If you are using something like node.fs: 'empty' replace it with resolve.fallback.fs: false.
+         * https://webpack.js.org/migrate/5/
+         */
+        config.resolve.fallback = {
+            fs: false
         };
-    }
 
-    // Export the plugin module
-    modulePath = modulePath.substr(0, modulePath.indexOf('.ts'));
-    const entryPointContents = `export * from '${modulePath}';`;
-
-    if (!options.preserveMainFile) {
-        patchFile(entryPointPath, entryPointContents);
-    }
+        // Make sure we are producing a single bundle
+        delete config.entry['polyfills'];
+        delete config.optimization.runtimeChunk;
+        delete config.optimization.splitChunks;
+    
+        if (!options.precalculateRem) {
+            delete config.entry['styles'];
+        }
+    
+        // List the external libraries which will be provided by vcd
+        config.externals = [
+            ...(options.ignoreDefaultExternals ? [] : defaultExternals.common),
+            ...(!options.enableRuntimeDependecyManagement && !options.ignoreDefaultExternals ? defaultExternals['9.7-10.0'] : []),
+            ...extractExternalRegExps(options.externalLibs),
+        ];
+
+        // preserve path to entry point
+        // so that we can clear use it within `run` method to clear that file
+        const entryPointPath = config.entry['main'][0];
+        const entryPointOriginalContent = fs.readFileSync(entryPointPath, 'utf-8');
+
+        // Load manifest file
+        const manifestJsonPath = path.join(`${config.context}/src/public`, 'manifest.json');
+        const manifest: ExtensionManifest = JSON.parse(fs.readFileSync(manifestJsonPath, 'utf-8'));
+        const manifestOriginalContent: string = fs.readFileSync(manifestJsonPath, 'utf-8');
+        
+        // Patch the main.ts file to point to the plugin which will be compiled
+        let [modulePath, moduleName] = options.modulePath.split('#');
+
+        if (options.enableRuntimeDependecyManagement) {
```

Comment:
Please fix the type enableRuntimeDependecyManagement -> enableRuntimeDependencyManagement